### PR TITLE
Remove XDebug

### PR DESCRIPTION
**Pull request by dhrrgn**

### CODE REVIEW

**Comment by shadowhand** [comment]
@dhrrgn has a conflict.

File: Dockerfile [hunk 1]
```diff
@@ -28,3 +28,6 @@ RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
 
 # Add additional development settings
 COPY xdebug.ini "$PHP_INI_DIR/conf.d/90-xdebug.ini"
+
+# Disable Xdebug by default
+RUN sed -i -e 's/^zend_extension/;zend_extension/g' /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
```
Comment (shadowhand) [inline Dockerfile]: Perhaps a better alternative would be to remove `xdebug` completely. We can always restore it by looking back at this PR.

Comment (dhrrgn) [reply Dockerfile]: Ya, just going to do that.

**Comment by dhrrgn** [comment]
@shadowhand Updated and fixed merge conflict

File: Dockerfile [hunk 2]
```diff
@@ -17,14 +17,9 @@ RUN /root/install-composer.sh \
 
 # Add development PHP extensions
 RUN apk add --no-cache --virtual .build-deps $PHPIZE_DEPS \
-    && pecl install xdebug-2.7.1 \
-    && docker-php-ext-enable xdebug \
     && apk del .build-deps
```
Comment (shadowhand) [inline Dockerfile]: These two lines can be removed too.

Comment (dhrrgn) [reply Dockerfile]: Oh, thought those were required.  Thanks.  Removed.

**Pull request merged**